docs/operate/get-started/setup-micro/_index.md

```diff
@@ -0,0 +1,404 @@
+   ```
+
+   To use any of the [example modules](https://github.com/viamrobotics/micro-rdk/blob/main/examples/modular-drivers/README.md#example-modules) provided by Viam, use this line, specifying the correct Micro-RDK version that matches your development environment:
+
+   ```toml {class="line-numbers linkable-line-numbers" data-line="3"}
+   [dependencies]
+   ...
+   micro-rdk-modular-driver-example = { git = "https://github.com/viamrobotics/micro-rdk.git", rev = "v0.5.0", package = "micro-rdk-modular-driver-example", features = ["esp32"] }
+   ```
+
+   {{% /tab %}}
+   {{< /tabs >}}
+
+1. Compile the project:
+
+   ```sh { class="command-line" data-prompt="$"}
+   make build-esp32-bin
+   ```
+
+   The first build may be fairly time consuming, as ESP-IDF must be cloned and built, and all dependent Rust crates must be fetched and built.
+   Subsequent builds will be faster.
+
+### Flash your ESP32
+
+Upload the generated firmware to your ESP32:
+
+1. Use a data cable to connect your ESP32 board to a USB port on your development machine.
+
+1. Run the following command to flash the firmware to your ESP32:
+
+   ```sh { class="command-line" data-prompt="$"}
+   make flash-esp32-bin
+   ```
+
+   When prompted, select the serial port that your ESP32 is connected to through a data cable.
+
+   If the flash is successful, you will retain a serial connection to the board until you press `Ctrl-C`.
+   While the serial connection is live, you can also restart the currently flashed image with `Ctrl-R`.
+
+1. Navigate to your new machine's page on the [Viam app](https://app.viam.com).
+   If successful, the status indicator should turn green and show **Live**.
+
+### Configure and test your machine
+
+You can now configure the models you included in your firmware and test them:
+
+1. Navigate to your machine's page in the [Viam app](https://app.viam.com).
+
+1. From the **CONFIGURE** tab, click **JSON** mode.
+   Micro-RDK components and services must be configured in JSON.
+
+1. Add your components and services to the machine configuration.
+   If you are using one of the example modules, find configuration details in the [example modules README](https://github.com/viamrobotics/micro-rdk/blob/main/examples/modular-drivers/README.md#example-modules).
+   For example, if you want to use a free heap sensor, your configuration could look like this:
+
+   ```json
+   {
+     "components": [
+       {
+         "name": "my-free-heap-sensor",
+         "api": "rdk:component:sensor",
+         "model": "free-heap",
+         "attributes": {}
+       }
+     ]
+   }
+   ```
+
+1. Click **Builder** mode and find the configuration card of the component you want to test.
+
+1. Click to open the **Test** section of the card and use the interface to test the component.
+
+1. If you make changes to your modules, rerun the build and flash steps to update the firmware and test the new version on your ESP32.
```

Comment:
```suggestion
1. If you make changes to your module implementation, rerun the build and flash steps to update the firmware and test the new version on your ESP32.
```
[s] unless we clarify, I suspect some users might misunderstand and think that they need to re-flash even if they only changed the configuration (or maybe I'm misunderstanding!)

Reply:
How about this?
```suggestion
1. If you make changes to your module code, rerun the build and flash steps to update the firmware and test the new version on your ESP32.
```